build.gradle

```diff
@@ -5,6 +5,12 @@ buildscript {
         compose_version = '1.2.1'
         activity_compose_version = '1.6.0'
         core_ktx_version = '1.9.0'
+
+        nav_version = "2.5.3"
+        navigation_compose = "androidx.navigation:navigation-compose:$nav_version"
```

Comment:
Some convention (the one we stick to) is that you put versions here but the implementation paths "androidx.navigation:navigation-compose"  is hardcoded in app build.gradle (where it was in previous commit). The idea behind extracting versions is that you will use same version for multiple dependencies and that's way we store them in variable, to make increasing versions easier (we just have to change variable instead of changing every dependency.  This doesn't hold for paths because they probably won't change and they are unique. That's why we just hardcode them. 
You don't have to change this now, you can just modify it when you turn in your next homework.